src/interfaces/minter.cairo

```diff
@@ -33,15 +33,23 @@ namespace ICarbonableMinter:
     func max_supply_for_mint() -> (max_supply_for_mint : Uint256):
     end
 
+    func merkle_root() -> (root : felt):
```

Comment:
I suggest we use a more explicit name like `whitelist_merkle_root`

Reply:
Done in [98223d1](https://github.com/Carbonable/carbonable-starknet/pull/22/commits/98223d10be7c60a44e759013f91cd29b433bbaeb)